contrib/update-my-copyright.pl

```diff
@@ -120,18 +120,18 @@ sub quiet_print {
 $year += 1900;
 quiet_print "==> This year: $year\n";
 
-# Find the top-level OMPI source tree dir
+# Find the top-level repository source tree dir
 my $start = cwd();
 my $top = $start;
-while (! -f "$top/Makefile.ompi-rules") {
+while (! -f "$top/VERSION" && ! -f "$top/HACKING") {
```

Comment:
Do you want to look for `.git`?  That would make this work in any git repo (i.e., be even less OMPI-centric).